rust/src/dcerpc/dcerpc.rs

```diff
@@ -1818,216 +1818,6 @@ mod tests {
         );
     }
 
-    #[test]
```

Comment:
> Given that these code paths are already covered by tests somewhere

Where is this somewhere ?

Reply:
Suricata verify

Reply:
you can see the Codecov report that this code is almost fully covered: https://app.codecov.io/gh/OISF/suricata/pull/11793/blob/rust/src/dcerpc/dcerpc.rs

Reply:
I think in the end, these tests are good to be removed, but I would like a bit more history to know which SV tests cover this, what is the feature being tested, etc...

Reply:
ok. I'll try to find all those and add in the commit message. Thank you!